webcompat/static/js/lib/bugform.js

```diff
@@ -155,6 +156,18 @@ function BugForm() {
     img.src = dataURI;
   };
 
+  // Event handler for image file drop; purpose to detect multiple drops
+  this.handleImageDrop = function() {
+    var imageFilled = false;
+    if ($(".wc-UploadForm-label.is-hidden").length) {
+      imageFilled = true;
+    }
+    if (imageFilled) {
+      // if 2nd drag and drop make wrapper hidden to make remove button available
+      $(".wc-UploadForm-wrapper").addClass("is-hidden");
```

Comment:
I don't understand how this is supposed to work, can you explain it to me? Why does adding a `is-hidden` class to this wrapper div solve the problem?

It seems like in the `drop` handler we want to just do the same as we do for the change event on the file input -- call `checkImageTypeValidity`, right (we need to make sure we send it the right data from the event, or `dataTransfer` object or something, it's been a while since I've messed with DnD....)?

(we also want to prevent the default browser action, otherwise we end up navigating to the image. try to drop a new image 3 times.... why 3 and not 2 or 1... not sure. 🤔